src/service/video-manager-impl.js

```diff
@@ -118,6 +124,38 @@ export class VideoManager {
 
     /** @private {?VideoEntry} */
     this.dockedVideo_ = null;
+
+    /** @private {?./timer-impl.Timer} */
+    this.timer_ = null;
+
+    /** @private {?Function} */
+    this.boundSecondsPlaying_ = null;
+
+    const isAnalyticsEnabled = getMode().test ?
```

Comment:
I guess if we can't detect whether they have video-analytics or not, this optimization won't really help much ( I expect vast majority of AMP pages have some sort of analytics anyway). let's replace with a TODO + GitHub issue to expose analytics config to components somehow to enable optimizations we want.